keep-ui/app/rules/CorrelationSidebar/index.tsx

```diff
@@ -7,8 +7,8 @@ import { RuleGroupType } from "react-querybuilder";
 export const DEFAULT_CORRELATION_FORM_VALUES: CorrelationForm = {
   name: "",
   description: "",
-  timeAmount: 5,
-  timeUnit: "minutes",
+  timeAmount: 24,
+  timeUnit: "hours",
```

Comment:
its still show it in seconds (86400) which looks bit odd

Reply:
The problem here is that the backend stores seconds. I decided not to add a field right now to store the unit right now, but if you think it's important - I'll do

Reply:
Done